fzf-notes

```diff
@@ -188,6 +192,10 @@ function interact_with_notebook() {
 
 declare -A globals
 globals[NEW_FILE_CREATION_OPTION]="CREATE_NEW_FILE"
+
+# File extension to use if none entered for CREATE_NEW_FILE.
+# Set to "md" to default to creating markdown docs.
+globals[NEW_FILE_CREATION_EXTENSION]=""
```

Comment:
Since this is an option that's being configured, it should go in the config file. It's a YAML file, and this option can be easily added and then parsed out with yq, just like the other configuration options are.